UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.FulfilmentService/Program.cs

```diff
@@ -96,6 +133,9 @@ private static HostBuilder BuildHostConfiguration()
                  services.AddHttpClient<IFileShareServiceClient, FileShareServiceClient>(client =>
                     client.BaseAddress = new Uri(ConfigurationBuilder["FileShareService:BaseUrl"])
                  );
+                 services.AddHttpClient<IFileShareServiceClient, FileShareServiceClient>(client =>
```

Comment:
The `HttpClient` has already been configured for `IFileShareServiceClient` on line 133, shouldn't this be combined with that one?

Reply:
As per the suggestion i have made changes.